src/components/shared/SelectRecipients/ProfileInList.tsx

```diff
@@ -32,7 +34,7 @@ export function ProfileInList(props: ProfileInListProps) {
     const classes = useStylesExtends(useStyle(), props)
     const profile = props.item
     const name = profile.nickname || profile.identifier.userId
-    const secondary = profile.linkedPersona?.fingerprint ? profile.linkedPersona?.fingerprint.toLowerCase() : undefined
+    const secondary = profile.linkedPersona?.fingerprint ? profile.linkedPersona?.fingerprint.toLowerCase() : ''
```

Comment:
Highlighter textToHighlight: string

Reply:
does it affect the secondary slot of UI?

Reply:
No, it results in the same.

Reply:
secondary UI will be highlighted both.